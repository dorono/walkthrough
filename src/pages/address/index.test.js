import React from 'react';
import {shallow} from 'enzyme';
import {withRouter} from 'react-router-dom';

import Address from './index';

const AddressElement = React.createElement(withRouter(<Address />));

describe('Address Page', () => {
    it('should render without errors', () => {
        shallow(AddressElement);
    });

    it('should match snapshot', () => {
        expect(shallow(AddressElement)).toMatchSnapshot();
    });
});
