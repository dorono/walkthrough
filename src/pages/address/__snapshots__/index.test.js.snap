// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Address Page should match snapshot 1`] = `
<Route>
  <Component />
</Route>
`;
